config/core/test-configs.yaml

```
@@ -1318,6 +1318,21 @@ device_types:
     params: &uefi_i386
       bios_url: 'http://storage.kernelci.org/images/uefi/edk2-stable202005/i386/OVMF.fd'
 
+  qemu_mips-malta:
+    base_name: qemu
+    mach: qemu
+    arch: mips
+    boot_method: qemu
+    context:
+      arch: mipsel
+      guestfs_interface: 'ide'
+      machine: 'malta'
+      memory: 256
+      model: 'model=pcnet'
+      no_kvm: True
+    filters:
+      - passlist: {defconfig: ['malta_kvm_guest_defconfig']}
```

If `malta_defconfig` works then should the filter be updated accordingly?
```suggestion
      - passlist: {defconfig: ['malta_defconfig']}
```